great_expectations/core/expectation_suite.py

```diff
@@ -53,14 +65,14 @@ class ExpectationSuite(SerializableDictDot):
 
     def __init__(
         self,
-        expectation_suite_name,
-        data_context=None,
-        expectations=None,
-        evaluation_parameters=None,
-        data_asset_type=None,
-        execution_engine_type=None,
-        meta=None,
-        ge_cloud_id=None,
+        expectation_suite_name: str,
```

Comment:
Thanks for adding types. If you know the shape of the `dict`s you can annotate the keys and values like `Dict[str, str]`. I find this adds a bit of value since the type checker and check that you are using the right type of keys and can check that you do with the values.

Reply:
Good point! I think both meta and evaluation_parameters are a bit open ended as far as what can go in the dict. I'll have to do a bit more spelunking to confirm or deny - I'll leave as `dict` for this PR and follow up in a subsequent PR as I continue to clean up these tests.